leaderboard/interface.go

```diff
@@ -0,0 +1,35 @@
+package leaderboard
+
+import (
+	"context"
+
+	"github.com/topfreegames/podium/leaderboard/model"
+)
+
+// Leaderboard holds functional struct inside leaderboard and your dependecies to be executed
+type Leaderboard interface {
+	Healthcheck(ctx context.Context) error
+
+	// IncrementMemberScore(ctx context.Context, leaderboard string, member string, increment int, scoreTTL string) (*model.Member, error)
+	// SetMemberScore(ctx context.Context, leadeboard, member string, score int64, prevRank bool, scoreTTL string) (*model.Member, error)
+	// SetMembersScore(ctx context.Context, leaderboardID string, members []*model.Member, prevRank bool, scoreTTL string) error
```

Comment:
Those commented methods are commented because you didn't implement them yet and you will uncomment as you implement them?

Reply:
Yes